EssentialEstablishmentGenerator/Dictionary/floraData.js

```diff
@@ -26,5 +26,24 @@ setup.flora = {
     'tree': [
       'apple', 'almond', 'apricot', 'cherry', 'orange', 'lemon', 'lime', 'elderberry', 'fig', 'mango', 'olive', 'mulberry', 'peach', 'plum', 'pomegranate', 'walnut', 'chestnut'
     ]
+  },
+  'vegetable': {
+    // list of common vegetables, plural
+    'vegetableP': [
```

Comment:
Are vegetableP and vegetableS intentionally discrete arrays? i.e. is there ever a case when you'll need to access the same vegetable ("I grew the biggest [vegetableS] last year, but this year's crop of [corresponding vegetableP] isn't doing so well...")? Because if so, it might be wiser to store them as objects.

Reply:
I don't particularly expect to use the plural and singular versions of these in a single generation, but I can move all of it to objects, I think